src/app/pages/question/question.component.ts

```diff
@@ -38,6 +46,8 @@ export class QuestionComponent implements OnInit, CanDeactivate<CanComponentDeac
   modalWindowChoice$ = new BehaviorSubject<boolean>(false);
   modalWindowData$ = new BehaviorSubject<ModalWindowModel>({ page: '', title: '', text: '', link: '' });
 
+  private nextRoute: string = '';
```

Comment:
let's move private property after inputs. So the order you should follow: inputs, private values, public values, getters, constructor, CanDeactivate, ngOnInit, public methods, private methods.
Apply such order for this file